resources/public/api/conf/2.0/schemas/historic_uk_non_fhl_period_summary_list/response.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "submissions": {
+      "type": "array",
+      "items": [
```

Comment:
![Screenshot 2022-09-09 at 10 26 07](https://user-images.githubusercontent.com/108878081/189318225-6619065f-b907-4680-88d9-2b1c29130070.png)

description of the items are not showing, can remove the []